### PR TITLE
Update logic for multiselecting.

### PR DESCRIPTION
**Pull request by doncung**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @doncung has signed the CLA. Thanks for the pull request!

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import com.box.androidsdk.content.requests.BoxResponse;
 import com.box.androidsdk.content.utils.SdkUtils;
 
+import java.io.Serializable;
```
Comment (brianytang) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java]: are we using this?

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxBrowseFragment.java [hunk 2]
```diff
@@ -862,12 +867,26 @@ void toggle(BoxItem boxItem){
...
+            mItemAdapter = new WeakReference<BoxItemAdapter>(adapter);
+        }
+
+
         boolean isMultiSelecting(){
             return mIsMultiSelecting;
         }
 
-        void setMultiSelecting(boolean enabled){
+        public void setMultiSelecting(boolean enabled){
```
Comment (brianytang) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxBrowseFragment.java]: setEnabled

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxBrowseFragment.java [hunk 3]
```diff
@@ -862,12 +867,26 @@ void toggle(BoxItem boxItem){
             handleItemSelected(boxItem, wasSelected, this);
         }
 
+        void setItemAdapter(BoxItemAdapter adapter){
+            mItemAdapter = new WeakReference<BoxItemAdapter>(adapter);
+        }
+
+
         boolean isMultiSelecting(){
```
Comment (brianytang) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxBrowseFragment.java]: isEnabled

**Comment by brianytang** [comment]
+1

**Pull request merged**